### PR TITLE
fix(settings-modal): fixed modal style

### PR DESCRIPTION
**Pull request by ThomBos**
**What this PR does** 📖
Fixes settings' modal style

**Which issue(s) this PR fixes** 🔨
AP-1765

**Special notes for reviewers** 🗒️

**Additional comments** 🎤

### CODE REVIEW

**Comment by ThomBos** [comment]
forgot to report this depends on #3514

**Comment by stavares843** [comment]
https://github.com/Satellite-im/Core-PWA/pull/3514 is merged 🎉

File: components/ui/SimpleScroll/SimpleScroll.vue [hunk 1]
```diff
@@ -30,6 +30,11 @@ export default Vue.extend({
       default: '',
       required: false,
     },
+    invisibleScrollbar: {
+      type: Boolean,
+      default: false,
+      required: false,
+    },
```
Comment (molimauro) [inline components/ui/SimpleScroll/SimpleScroll.vue:37]: what is this?

Comment (ThomBos) [reply components/ui/SimpleScroll/SimpleScroll.vue:37]: this allows us to program when it's time to show the scrollbar. 
with the select we have a little graphic issue related to the scrollbar: it keeps the space even if the scrollbar isn't necessary (due to webkit styling, even removing paddings that space remains)

File: components/interactables/Select/Select.html [hunk 2]
```diff
@@ -23,12 +23,17 @@
       {{ getSelectLabel }}
     </div>
     <div class="items">
-      <UiSimpleScroll scrollMode="vertical" scrollShow="scroll">
+      <UiSimpleScroll
+        scrollMode="vertical"
+        scrollShow="always"
+        :invisibleScrollbar="options.length < 7"
```
Comment (ThomBos) [inline components/interactables/Select/Select.html:29]: check this out

Comment (ThomBos) [reply components/interactables/Select/Select.html:29]: or it would be like 50/80px tall 😓

**Comment by molimauro** [comment]
@ThomBos we merged a PR from Joe regarding the select fix, is the invisible thing still needed?

**Comment by ThomBos** [comment]
> @ThomBos we merged a PR from Joe regarding the select fix, is the invisible thing still needed?

it depends if in the fix there's the scrollbar component... i'm checking it

**Comment by ThomBos** [comment]
> @ThomBos we merged a PR from Joe regarding the select fix, is the invisible thing still needed?

yes, need until we change select component!
if the new component is going to use "simplescrollbar" too, we'll keep that invisible option 

that invisible option wouldn't be necessary just if there was a way to tell the scrollbar not to keeping those 5px padding (scrollbar-width/height)

**Comment by josephmcg** [comment]
you can remove any edits to Select in your rebase

File: components/ui/SimpleScroll/SimpleScroll.vue (hunk 1 above)
Comment (molimauro) [reply components/ui/SimpleScroll/SimpleScroll.vue:37]: So is this still necessary after the rework?

**Review by molimauro — CHANGES_REQUESTED** [review]

**Review by phillsatellite — APPROVED** [review]

**Review by molimauro — APPROVED** [review]

**Review by stavares843 — APPROVED** [review]

**Pull request merged**